src/emailcontact.cpp

```diff
@@ -0,0 +1,66 @@
+#include "emailcontact.h"
```

Comment:
nitpick: missing license header in various .cpp files

Reply:
done